### ISSUE TITLE
Using databaseType does not work

### ISSUE THREAD

**Issue by mvilrokx** [promptfoo/promptfoo#1446]
**Describe the bug**
The feature `databaseType` described [here](https://www.promptfoo.dev/docs/guides/text-to-sql-evaluation/) does not work as intended. You can put any value you want against `databaseType`, that value will never be used.

**To Reproduce**
Use the following assert, it will ALWAYS use MySQL (which is the default one):

```
    assert:
      - type: contains-sql
        value:
          databaseType: 'Sqlite'
```

**Expected behavior**
promptfoo should pick up the configured `databaseType`.

**System information (please complete the following information):**

- Promptfoo version: `0.78.0`

**Additional context**
This could be a documentation issue because the documentation states everywhere to use `databaseType` but [the code clearly expects `database`](https://github.com/promptfoo/promptfoo/blob/9b7935854002e9f540ff857612faf10ef758577e/src/assertions.ts#L210). So either that line has to be changed to `databaseType = value.databaseType || 'MySQL';` OR the documentation needs to be corrected. I presume the former is more "backwards compatible".

I have tried with:
```
    assert:
      - type: contains-sql
        value:
          database: 'Sqlite'
```
and that works fine.

**Comment by typpo** [comment]
Fixed in #1451

**Comment by mldangelo** [comment]
@mvilrokx this should be fixed in [0.78.1](https://github.com/promptfoo/promptfoo/releases/tag/0.78.1). Thank you so much for taking the time to report this issue.
